src/actions/userActions.js

```diff
@@ -130,3 +135,39 @@ export const getProfile = () => dispatch => {
       dispatch({ type: LOGOUT_USER, payload: false });
     });
 };
+
+export const sendResetLink = (userDetails) => dispatch => {
+  toast.dismiss();
+  dispatch({ type: SEND_RESET_LINK_INITIATED, payload: true });
+  return axiosInstance
+    .post('/api/users/password_reset/', userDetails)
+    .then(response => {
+      dispatch({ type: SEND_RESET_LINK_SUCCESS, payload: true });
+      toast.success(response.data.user.message, { autoClose: 3500, hideProgressBar: true });
+    })
+    .catch(() => {
+      dispatch({ type: SEND_RESET_LINK_ERROR, payload: 'Please enter a valid email' });
+      toast.error('Please enter a valid email', { autoClose: 3500, hideProgressBar: true });
+    });
+};
+
+export const resetPassword = (passwordDetails) => dispatch => {
+  toast.dismiss();
+  axiosInstance
+    .put('/api/users/password_update/', passwordDetails)
+    .then(response => {
+      dispatch({ type: RESET_PASSWORD_SUCCESS, payload: true });
+      toast.success(response.data.user.message, { autoClose: 3500, hideProgressBar: true });
+      localStorage.removeItem('auth_token');
+    })
+    .catch((error) => {
+      if (error.response) {
+        dispatch({ type: RESET_PASSWORD_ERROR, payload: error.response.data.errors[0] });
+        return toast.error(
+          error.response.data.errors[0],
+          { autoClose: 3500, hideProgressBar: true },
+        );
+      }
+      return toast.error('Error connection', { autoClose: 3500, hideProgressBar: true });
```

Comment:
Thank you for handling errors. However, I think you should look into changing` Error connection` to either `Connection error` or some other message.